src/base/components/Dashboard.js

```diff
@@ -0,0 +1,109 @@
+import React from 'react';
+import { connect } from "react-redux";
+
+class DashboardPage extends React.Component {
+
+  constructor(props) {
+    super(props);
+    this.state = {
+      sensorData: []
+    }
+  }
+
+  componentDidMount() {
+    setTimeout(() => {
+      this.setState({ sensorData: this.props.sensorData })
+    }, 3000);
+    document.getElementsByClassName('main')[0].style.width = "1300px";
+    document.getElementsByClassName('main')[0].style.maxWidth = "1300px";
+  }
+
+  render() {
+    let sensorData = this.state.sensorData;
+    let fetchedData = sensorData.length ? sensorData.map(item => item.payload).map(item => item.dataFeedElement) : [];
+
+    return (<div>
+      <span class="tool-tip" data-toggle="tooltip" data-placement="top" title="Coming">
+        <button disabled="disabled" class="btn btn-default">Add New Device</button>
+      </span>
+      <div>
+        <span class="tool-tip" data-toggle="tooltip" data-placement="top" title="Upgrade to premium. Coming">
+          <button disabled="disabled" class="btn btn-default">Group Management</button>
+        </span>
+      </div>
+      <table class="table table-bordered">
+        <thead>
+          <tr>
+            <th>
+              <div class="custom-control custom-checkbox">
+                <input type="checkbox" class="custom-control-input" id="tableDefaultCheck1" disabled="true" />
+              </div>
+            </th>
+            <th>ID </th>
+            <th>Name </th>
+            <th>Location </th>
+            <th>Access </th>
+            <th>Cost </th>
+            <th>Protocol </th>
+            <th>Role </th>
+            <th>Alerts </th>
+            <th>State </th>
+            <th>Last seen</th>
+            <th>Last measurement</th>
+            <th>Device battery measurement</th>
```

Comment:
Delete "measurement".

Reply:
Device battery measurement to Device battery ?

Reply:
See https://github.com/webRunes/WRIO-InternetOS/issues/1581#issuecomment-499443413
All must be as described

Reply:
updated